csunplugged/topics/content/en/sorting-networks/sorting-networks.yaml

```diff
@@ -0,0 +1,4 @@
+unit-plans:
+    - unit-plan/unit-plan.yaml
+
+icon: img/topics/sorting-network-kids.png
```

Comment:
similar to in resources, should we have a "thumbnail.png" file for each topic? Conscious of this happening in future:

Someone is updating a lesson and see that an image is no longer relevant:
> "I'm deleting that image from this lesson, I'll be a tidy kiwi and delete it from the images folder too!"

10 mins later the same someone is reviewing their work and go to the /topics page:

> "hey! where did the cute little icon go?! Oh no, I must have deleted it because it didn't have a filename that suggested to me that I shouldn't delete it ever! 😢 "